release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -3357,6 +3357,18 @@ You can view the container images in this release by running the following comma
 $ oc adm release info 4.10.30 --pullspecs
 ----
 
+[id="ocp-4-10-30-bug-fixes"]
+==== Bug fixes
+
+* Previously, the functionality of Bond-CNI was limited to only active-backup mode. With this update, the bonding modes supported are:
+
+** `balance-rr - 0`
```

Comment:
Question. In this module, the modes are formatted like this:
`balance-rr` - 0
`active-backup` - 1
`balance-xor` - 2

Module: https://github.com/openshift/openshift-docs/blob/main/modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc
Docs: https://docs.openshift.com/container-platform/4.11/networking/hardware_networks/using-pod-level-bonding.html#nw-sriov-cfg-creating-bond-network-attachment-definition_using-pod-level-bonding

So I wasn't sure which one was right

Reply:
ooo thanks for the comment. Need to fix 4.11 note then too

Reply:
updated!